lib/registries/addon/components/sharing-icons/sharing-anchor/component.ts

```diff
@@ -0,0 +1,28 @@
+import { tagName } from '@ember-decorators/component';
+import { action } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import Analytics from 'ember-osf-web/services/analytics';
+import defaultTo from 'ember-osf-web/utils/default-to';
+import layout from './template';
+
+@tagName('')
+export default class SharingAnchor extends Component {
+    layout = layout;
```

Comment:
[Prefer the layout decorator](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/decorators/component.ts#L36)